engine/swarm/services.md

```diff
@@ -70,6 +70,86 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your local registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+When you don't specify a tag, you get the latest version of an image, because
+Docker appends the `latest` tag. The following two commands are equivalent:
+
+```bash
+$ docker service create --name="myservice" ubuntu
+
+$ docker service create --name="myservice" ubuntu:latest
+```
+
+You can specify any tag which has been pushed to Docker Hub or your private
+registry. For instance, some of the tags available for the `ubuntu` image are
+`latest`, `devel`, `yakkety`, and `16.04`. The image's tag name is resolved to
+the specific digest it points to at the time of service creation. As an
+alternative, you can specify the digest directly, and no digest resolution
+occurs.
```

Comment:
Can we explicitly show what an image string looks like, after resolution to digest? Such as `ubuntu:latest@sha256:345kjhfkjaslf...`?